bridge-balance-difference/README.md

```diff
@@ -0,0 +1,26 @@
+# Bridge Balance Difference Bot
+
+## Description
+
+Detects if two sides of the bridge show significant balance difference
+
+## Supported Chains
+
+- Ethereum
+- Optimism
+- Binance Smart Chain
+- Polygon
+- Fantom
+- Arbitrum
+- Avalanche
+
+## Alerts
+
+- BRIDGE-BALANCE-DIFFERENCE
+  - Fired when two sides of the bridge show significant balance difference
+  - Severity is always set to "high"
+  - Type is always set to "suspicious"
+
+## Test Data
+
+The bot behaviour can verified with the provided unit tests
```

Comment:
you already know what I will ask here :)

Reply:
It will be hard to provide block/transaction range because we monitor 2 chains and we cannot get what was the last block on chain B when chain A had X blocks. We would have to significantly modify the bot to be able to test